Lib/axisregistry/data/round.textproto

```diff
@@ -0,0 +1,17 @@
+# SOFT based on internal project
+tag: "ROND"
+display_name: "Round"
+min_value: 0
+max_value: 100
+default_value: 0
+precision: 0
+fallback {
+  name: "Default"
+  value: 0
+}
+fallback_only: false
+illustration_url: "round.svg"
+description:
+  "Adjust terminals to become more and more rounded."
```

Comment:
I think that we're landing on a decision to make this axis more general purpose and the language here should change accordingly.  If there is consensus around this, then we should remove references to terminals altogether and make this a general purpose definition.

Thoughts @vv-monsalve?